### ISSUE TITLE
Medusa installed with apt produces error message

### ISSUE THREAD

**Issue by markkutikkanen** [thelastpickle/cassandra-medusa#201]
I didn't want to go configuring python environment and dependencies manually so I was happy to see you provide an APT repo. https://github.com/thelastpickle/cassandra-medusa/wiki/Installation#debian-packages

I followed the instructions and there was no errors.

However after installing any `medusa` invocation produces error:
```
~$ medusa
Traceback (most recent call last):
  File "/usr/bin/medusa", line 6, in <module>
    from pkg_resources import load_entry_point
ImportError: No module named 'pkg_resources'
```

The environment is Ubuntu 16.04.3

**Comment by arodrime** [comment]
Being discussed on slack here: https://the-asf.slack.com/archives/C0111B5KADA/p1599476136004200

**Comment by vinovarov** [comment]
Exactly same error on Debian 9.13, no googled workaround like re-installing python setuptools helped.

Would be great to have a working deb package as we would have to deploy Medusa (if we'll be able to adopt it).

**Comment by vinovarov** [comment]
Fix from https://github.com/thelastpickle/cassandra-medusa/issues/208 seems to work for apt-installed Medusa too, at least for me:

```
apt install python3-pip -y
pip3 install ssh2-python==0.19.0
```

Please confirm somebody.

**Comment by masterzen** [comment]
Go the same issue. BTW 0.7.0 is also affected.
I believe this is an issue with the virtualenv that is packaged, but I couldn't find what was wrong or how to heal it.

**Comment by masterzen** [comment]
A bit more information: the package works on a bionic 18.04 machine, but not on a xenial 16.04 machine. I think the reason is because the package is built in a 18.04 container, so assuming a python 3.8 environment.
I was able to rebuild a xenial package (but that required a few changes to the build system).

**Comment by masterzen** [comment]
I've started working on it @arodrime. I'm waiting for the packages to be built to test them and if that's working find, I'll create a PR.

**Comment by arodrime** [comment]
@masterzen that's amazing, thanks for your time! I'll do my best to review your request quickly once it's out and probably cut a release soon-ish!

**Comment by arodrime** [comment]
Closing this issue 'as fixed' in #213. Thanks @masterzen!
